src/Package/Impl/Options/R/RToolsOptionsPage.cs

```diff
@@ -38,7 +38,7 @@ public class RToolsOptionsPage : DialogPage {
         [CustomLocDisplayName("Settings_CranMirror")]
         [LocDescription("Settings_CranMirror_Description")]
         [TypeConverter(typeof(CranMirrorTypeConverter))]
-        [DefaultValue("0-Cloud [https]")]
+        [DefaultValue(null)]
```

Comment:
Then it will use whatever is the default for that R implementation. On vanilla R, if you ask, it says `@CRAN@`, which is a special value that package fetching code expands to "http://CRAN.R-project.org" when it detects it. I think the point of that indirection is so that if you use the default, new versions of R can change it, and that change will be automatically applied.